src/test/groovy/graphql/GraphQLTest.groovy

```diff
@@ -45,17 +47,19 @@ class GraphQLTest extends Specification {
                 .field(
                 newFieldDefinition()
                         .name("id")
+                        .dataFetcher(new MapDataFetcher("id"))
```

Comment:
I just updated the README on this branch to fix the mistake you found. The README also explains what the data fetchers to. They have very discrete, specific functionality.